.prettier.config.js

```diff
@@ -1,35 +1,35 @@
-'use strict';
+"use strict";
 
 module.exports = {
-  arrowParens: 'always',
+  arrowParens: "always",
   bracketSpacing: true,
-  endOfLine: 'lf',
+  endOfLine: "lf",
   printWidth: 80,
-  quoteProps: 'consistent',
+  quoteProps: "consistent",
   semi: true,
   singleQuote: true,
   tabWidth: 2,
-  trailingComma: 'es5',
+  trailingComma: "es5",
   useTabs: false,
   overrides: [
     {
-      files: ['.eslintrc', '.prettierrc', '.renovaterc.json', '*.json'],
-      options: { parser: 'json' },
+      files: [".eslintrc", ".prettierrc", ".renovaterc.json", "*.json"],
```

Comment:
Codacy has a fix for the issue: [Replace `".eslintrc",·".prettierrc",·".renovaterc.json",·"*.json"` with `'.eslintrc',·'.prettierrc',·'.renovaterc.json',·'*.json'`](https://app.codacy.com/gh/openinf/openinf.github.io/pullRequest?prid=9929905)
```suggestion
      files: ['.eslintrc', '.prettierrc', '.renovaterc.json', '*.json'],
```